es_ES/cookbook/lists/mixed-list.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: page
+title: "Creando listas con elementos de diferentes tipos"
+permalink: /cookbook/lists/mixed-list/
+---
+
+A menudo necesitamos crear listas que muestren diferentes tipos de contenido. Por ejemplo, podríamos estar trabajando en una Lista que muestra un encabezado seguido de algunos elementos relacionados con el encabezado, seguido de otro encabezado, y así sucesivamente.
+
+¿Cómo crearíamos una estructura así con Flutter?
+
+## Instrucciones
+
+  1. Crea una fuente de datos con diferentes tipos de elementos
+  2. Convierte la fuente de datos en una Lista de Widgets
+
+## 1. Crea una fuente de datos con diferentes tipos de elementos
+
+### Tipos de Elementos
+
+Para representar diferentes tipos de elementos en una Lista, necesitaremos definir una clase para cada tipo de elemento.
+
+En este ejemplo, trabajaremos en una aplicación que muestra un encabezado seguido de cinco mensajes. Por lo tanto, crearemos tres clases: `ListItem`, `HeadingItem`, 
+y `MessageItem`.
+
+<!-- skip -->
+```dart
+// La clase base para los diferentes tipos de elementos que la Lista puede contener
+abstract class ListItem {}
+
+// Un ListItem que contiene datos para mostrar un encabezado
+class HeadingItem implements ListItem {
+  final String heading;
+
+  HeadingItem(this.heading);
+}
+
+// Un ListItem que contiene datos para mostrar un mensaje
+class MessageItem implements ListItem {
+  final String sender;
+  final String body;
+
+  MessageItem(this.sender, this.body);
+}
+```
+
+### Crea una Lista de Elementos
+
+La mayoría de las veces, buscamos datos de Internet o de una base de datos local y los convertimos en una lista de elementos.
+ 
+Para este ejemplo, generaremos una lista de elementos para trabajar. La lista contendrá un encabezado seguido de cinco mensajes. Limpie, repita.
+
+<!-- skip -->
+```dart
+final items = List<ListItem>.generate(
+  1200,
+  (i) => i % 6 == 0
+      ? HeadingItem("Encabezado $i")
+      : MessageItem("Remitente $i", "Cuerpo del Mensaje $i"),
+);
+```
+
+## 2. Convierta la fuente de datos en una Lista de Widgets
+
+Para poder convertir cada elemento en un Widget, emplearemos el constructor 
+[`ListView.builder`](https://docs.flutter.io/flutter/widgets/ListView/ListView.builder.html)
+.
+
+En general, queremos proporcionar una función `builder` que verifique con qué tipo de elemento estamos tratando, y devuelva el Widget apropiado para ese tipo de elemento.
+
+En este ejemplo, utilizar la palabra clave `is` para comprobar el tipo de elemento con el que estamos tratando puede ser útil. Es rápido, y automáticamente lanzará cada elemento al tipo apropiado. ¡Sin embargo, hay diferentes maneras de abordar este problema si usted prefiere otro patrón!
```

Comment:
utilizar --> utiliza  o usa 

también la oración no se si quedaría mejor así :

puede ser útil usar la palabra clave is para comprobar el tipo de elemento con el que estamos tratando

lo dejo a tu criterio

Reply:
Tenía la duda desde que lo traduje. Se escucha mejor la sugerencia. corregido. Gracias